src/index.js

```diff
@@ -17,12 +17,14 @@ const merge = require('lodash/merge');
 const path = require('path');
 const Chalk = require('chalk').constructor;
 const { diffImageToSnapshot } = require('./diff-snapshot');
+const { ResultTypes } = require('./comparator-result');
+const fs = require('fs');
 
 function updateSnapshotState(oldSnapshotState, newSnapshotState) {
   return merge({}, oldSnapshotState, newSnapshotState);
 }
 
-function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customDiffConfig = {}, noColors = false } = {}) {
+function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customDiffConfig = {}, noColors = false, cleanPassingDiffs = false } = {}) {
```

Comment:
I'm debating whether to even have this as an option and just always clean up passing diffs or to just default to `true`? @jking90 what do you think?

Reply:
I went with false to maintain existing functionality